asserts/model.go

```diff
@@ -256,13 +263,25 @@ func checkModelSnap(snap map[string]interface{}, grade ModelGrade) (*ModelSnap,
 		return nil, fmt.Errorf("presence of snap %q must be one of required|optional", name)
 	}
 
+	isClassic, err := checkOptionalBoolWhat(snap, "classic", what)
+	if err != nil {
+		return nil, err
+	}
+	if isClassic && !classic {
```

Comment:
please rename `classic` to `modelIsClassic`

Reply:
thanks, that makes things clearer indeed